tests/is/L42/perftests/PerfTests.java

```diff
@@ -0,0 +1,167 @@
+package is.L42.perftests;
+
+import static is.L42.tools.General.L;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileDescriptor;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import is.L42.common.Parse;
+import is.L42.platformSpecific.javaTranslation.Resources;
+import is.L42.tests.DeployAdamsTowel;
+import is.L42.top.CachedTop;
+import is.L42.top.Init;
+
+public class PerfTests {
+  
+  public static final PrintStream out, err;
+  
+  private static final boolean SAVE_RESULTS = true;
+  
+  private static final File programOutput;
+  
+  static {
+    PerfCounters.setEnabled(true);
+    try {
+      programOutput = File.createTempFile("l42testoutput", ".log");
+      PrintStream stream = new PrintStream(new FileOutputStream(programOutput));
+      System.setOut(stream);
+      System.setErr(stream);
+      out = new PrintStream(new FileOutputStream(FileDescriptor.out));
+      err = new PrintStream(new FileOutputStream(FileDescriptor.err));
+      } 
+    catch (Exception e) {
+      throw new Error(e);
+      }
+    }
+  
+  @AfterAll
+  public static void showLog() {
+    System.setOut(out);
+    System.setErr(err);
+    System.out.println("L42 Program output logged to: " + programOutput.getAbsolutePath());
+    }
+  
+  @Test 
```

Comment:
If you try-catch-error the exceptions in testRun, you can avoid the cutpasted throws.
You can then put @Test on the same line of the test name, and make a 2 line test instead of a 4 lines one.
This does becomes important when you have a lot of tests. Try to format them compact